src/App.jsx

```diff
@@ -1,5 +1,48 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+import { requestToken, instance } from './DataService/dataService';
+import { COOKIE } from './DataService/cookieService';
+import { findMyLocation } from './Helpers/geolocation';
+
 function App() {
-  return <div>Hello world!</div>;
+    const [resourceType, setResourceType] = useState('location/search/Bucharest');
+    const [location, setLocation] = useState('');
+    const [items, setItems] = useState([]);
+
+    async function tokenCheck() {
+        const token = (await requestToken()) || COOKIE.loadToken();
+        if (token) {
+            COOKIE.saveToken(token);
+        }
+    }
+
+    useEffect(async () => {
+        await tokenCheck();
+    }, []);
+
+    useEffect(async () => {
+        const res = await instance.get(`${resourceType}`);
+        console.log(res.data);
+
+        // setItems(res.data);
+    }, [resourceType]);
+
+    return (
+        <div>
+            <button onClick={() => setResourceType('location/search/Bucharest')}>Location</button>
+            <button onClick={() => setResourceType('observation/latest/100683506')}>
```

Comment:
hardcoded magic numbers should be stored in separate variables (please try to avoid hardcoded coords at all)

Reply:
done